Marlin/src/HAL/STM32F1/HAL.h

```diff
@@ -144,6 +144,17 @@
   #endif
 #endif
 
+#ifdef RS485_SERIAL_PORT
+  #if RS485_SERIAL_PORT == -1
+    #define RS485_SERIAL UsbSerial
+  #elif WITHIN(RS485_SERIAL_PORT, 1, NUM_UARTS)
+    #define RS485_SERIAL MSERIAL(RS485_SERIAL_PORT)
+  #else
+    #define RS485_SERIAL MSERIAL(1) // dummy port
+    static_assert(false, "RS485_SERIAL_PORT must be from 1 to " STRINGIFY(NUM_UARTS) ". You can also use -1 if the board supports Native USB.")
+  #endif
+#endif
```

Comment:
How does RS485 work in concert with the USB port?

Reply:
It doesn't. I more or less copied the other serial sections.